modules/tensor_mechanics/tests/recompute_radial_return/affine_plasticity.i

```diff
@@ -320,8 +320,8 @@
   #Preconditioned JFNK (default)
   solve_type = 'PJFNK'
   petsc_options = '-snes_ksp_ew'
-  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
-  petsc_options_value = 'lu superlu_dist'
+  petsc_options_iname = '-pc_type'
+  petsc_options_value = 'lu'
```

Comment:
Yea, i think this test is one element only.  The real reason i removed the superlu_dist was that one of my computers doesn't have it, so the test was failing all the time.